src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -196,4 +196,58 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
 
     Mockito.verify(mockResponse).sendRedirect("/chat/test_conversation");
   }
+  
+  @Test
+  public void testDoPost_ModifyMessageBoldItalics() throws IOException, ServletException{
+    Mockito.when(mockRequest.getRequestURI()).thenReturn("/chat/test_conversation");
+    Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
+
+    User fakeUser = new User(UUID.randomUUID(), "test_username", "password", Instant.now());
```

Comment:
Again, might have to update by adding a new field if Tishauna's pull request gets merged first.